packages/rspack-test-tools/tests/diagnosticsCases/builtins/html_syntax_error/stats.err

```diff
@@ -1,13 +1,6 @@
-ERROR in × HTML parsing error: Start tag seen without seeing a doctype first, expected "<!DOCTYPE html>"
-   ╭─[1:0]
- 1 │ <head>
-   · ──────
- 2 │   </something>
-   ╰────
-
 ERROR in × HTML parsing error: Stray end tag "something"
    ╭─[2:2]
- 1 │ <head>
+ 1 │ <!DOCTYPE html><head>
```

Comment:
This is a tricky way. When there is no doctype, the html5 doctype is added by default to prevent parsing errors.